pwa/src/services/auth.tsx

```diff
@@ -11,15 +11,12 @@ export const isBrowser = (): boolean => typeof window !== "undefined";
 export const handleLogin = async (data: IUnvalidatedUser, API: APIService) => {
   if (!isBrowser()) return;
 
-  return await API.Login.login(data).then((res) => {
-    API.setAuthentication(res.data.jwtToken);
-    navigate("/");
-  });
+  API.setAuthentication("0b4c72dd-3530-4f34-bfde-c7cc3e27c328");
```

Comment:
Is this an actual key? I assume it's not.

Maybe for that reason we could call it something like `"TEMPORARY-DUMMY-KEY"`.

This way, it's abundantly clear that it's not some mistakenly hardcoded key.